scripts/assemble_visualization_webpage.py

```diff
@@ -100,6 +100,9 @@ def get_gff_data(gff_dir):
     ] = "Uniprot predicted transmembrane regions"
 
     gff_map = {}
+    if not gff_dir:
```

Comment:
Is this testing for `None` or an empty string? One could make this check more explicit.

Reply:
I am basically checking for "didn't get a parameter for Gff" (no matter the reason: empty string, None, NULL).
The function should also handle the "none" case, but currently the empty string is the only way to pass over the command line.
(You see the kind of shorthand I am used from Perl)

(It's also the kind of corner case where PHP is going to bite you in the ass, because the string "0" evaluates to false there, not in Python.)

But how should I explicit this if it's not Pythonic?
`if gff_dir in (None, ''): ` ?

Reply:
I think that semantically using only `None` to represent "didn't get a parameter" makes the most sense. In that case, the check should simply be `is not None` (and not `!= None`).

But if we really want to handle all "falsy" values, then this should be fine.

Reply:
Currently, this parameters is passed as-is from the command line, so currently, it's mostly likely to be an empty string if set to "" in the config file.

Ideally, we should try playing with an argparser.

Reply:
And... [argparser done](https://github.com/cbg-ethz/V-pipe/pull/58/commits/8870fdaa931bbbf06546a5b3036edd0a205a35a0)